.gitignore

```diff
@@ -0,0 +1,2 @@
+bower_components
+bower.json
```

Comment:
I corrected, well I think over corrected.  I misunderstood the mail and push the bower_components in too. Now i know is just the bower.json, so you can now wich dependencies i used? Did I get it now?